src/tests/changelist.spec.ts

```diff
@@ -49,6 +53,23 @@ describe('changelist', () => {
 
     app = module.createNestApplication()
     await app.init()
+
+    const userData1 = createTestUser({
+      firstName: 'John',
+      lastName: 'Smith',
+    })
+    const userData2 = createTestUser({
+      firstName: 'Jane',
+      lastName: 'Doe',
+    })
+    const userData3 = createTestUser({
+      firstName: 'Will',
+      lastName: 'Duc',
+    })
+    const userRepository: Repository<User> = app.get(getRepositoryToken(User))
```

Comment:
You'l have to use the entitiyManager here

Reply:
yup